app/consumer-democracy/app.go

```diff
@@ -550,6 +554,7 @@ func New(
 		capabilitytypes.ModuleName,
 		authtypes.ModuleName,
 		banktypes.ModuleName,
+		consumertypes.ModuleName, // Note: consumer initiation before staking module
```

Comment:
@jstr1121 Why is this order necessary?

Reply:
Same here, InitGenesis shouldn't matter unless I'm missing something